Sources/Proxy/Representations/SlicedGeometryRepresentationProxy/index.js

```diff
@@ -0,0 +1,142 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkCutter from 'vtk.js/Sources/Filters/Core/Cutter';
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+import vtkPlane from 'vtk.js/Sources/Common/DataModel/Plane';
+
+import vtkAbstractRepresentationProxy from 'vtk.js/Sources/Proxy/Core/AbstractRepresentationProxy';
+
+// ----------------------------------------------------------------------------
+// vtkSlicedGeometryRepresentationProxy methods
+// ----------------------------------------------------------------------------
+
+function vtkSlicedGeometryRepresentationProxy(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkSlicedGeometryRepresentationProxy');
+
+  // Internals
+  model.plane = vtkPlane.newInstance();
+  model.cutter = vtkCutter.newInstance();
+  model.cutter.setCutFunction(model.plane);
+  model.mapper = vtkMapper.newInstance({
+    interpolateScalarsBeforeMapping: false,
+    useLookupTableScalarRange: false,
+    scalarVisibility: false,
+  });
+  model.actor = vtkActor.newInstance();
+  model.property = model.actor.getProperty();
+  model.property.setLighting(false);
```

Comment:
Good default? Why?

Reply:
would have been needed if I was using tubes (to get the thickness but not the depth). When only using lines, I'm not sure that matters, I'll test without it. I think it can be a good default anyway: when you look at a sliced geometry representation, it's usually just a contour, and you do not really care for lighting.